core/source/computed_field/computed_field_finite_element.cpp

```diff
@@ -1296,6 +1303,12 @@ cmzn_field *Computed_field_create_finite_element_internal(
 			/*number_of_source_fields*/0, NULL,
 			/*number_of_source_values*/0, NULL,
 			new Computed_field_finite_element(fe_field));
+		if (field && field->core)
+		{
+			Computed_field_finite_element *fieldFiniteElement=
+				static_cast<Computed_field_finite_element*>(field->core);
+			fieldFiniteElement->type = CMZN_FIELD_TYPE_FINITE_ELEMENT;
+		}
```

Comment:
This should check the value type of the FE_field and set type to FIELD_TYPE_STORED_STRING, or new FIELD_TYPE_STORED_MESH_LOCATION as appropriate, otherwise FIELD_TYPE_FINITE_ELEMENT. For the latter we need to work out how we write the integer/short and other value types of finite element; perhaps those should not be serialised yet.